### PR TITLE
Update kapacitor/README.md

### PR DESCRIPTION
**Pull request by AdamDang**
Delete dulicated word.
 Line 91: a Persistent Volume volume

### CODE REVIEW

**Comment by timstoop** [comment]
The reason that it's failing is that some of the requirements of the Chart.yaml have changed recently and this Chart is missing an `appVersion` key. You could add it and simply set it to the version of the image that's default. That would move this forward and fix it for future commits to this Chart as well.

**Comment by AdamDang** [comment]
@timstoop Thanks! I've add appVersion key.

File: stable/kapacitor/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,6 @@
 name: kapacitor
-version: 0.5.0
+version: 0.5.1
+appVersion: 0.5.1
```
Comment (sameersbn) [inline stable/kapacitor/Chart.yaml]: looking at https://github.com/AdamDang/charts/blob/e28a7a22a5d2af94f25a58ac64f7eecb76dde481/stable/kapacitor/values.yaml#L6, it appears like appVersion should be `1.4`

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/kapacitor/Chart.yaml (hunk 1 above)
Comment (AdamDang) [reply stable/kapacitor/Chart.yaml]: @sameersbn 
Thanks! I've modified the appVersion to 1.4-alpha.

File: stable/kapacitor/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name: kapacitor
-version: 0.5.0
+version: 0.5.1
+appVersion: 1.4-alpine
```
Comment (sameersbn) [inline stable/kapacitor/Chart.yaml]: you dont need to add the `-alpine` part. it should just be the version part

**Review by sameersbn — CHANGES_REQUESTED** [review]

**Review by sameersbn — APPROVED** [review]

**Pull request merged**